charts/core/README.md

```diff
@@ -235,6 +235,13 @@ Parameter | Description | Default | Notes
 `awsbilling.image.repository` | csp adapter image repository | `neuvector/neuvector-csp-adapter` | Follow AWS subscription instruction
 `awsbilling.image.tag` | csp adapter image tag | `1.0.0` | Follow AWS subscription instruction
 `awsbilling.image.imagePullPolicy` | csp adapter image pull policy | `IfNotPresent` | Follow AWS subscription instruction
+`global.azurebilling.enabled` | If true, install Azure billing csp adapter | `false` | **Note**: default admin user is disabled when global.azurebilling enabled, use configmap to create admin-role user to manage NeuVector deployment.
+`global.azurebilling.annotations` | Annotaion for csp adapter  | `nil` | Follow Azure subscription instruction
+`global.azurebilling.imagePullSecrets` | Pull secret for csp adapter image | `nil` | Follow Azure subscription instruction
+`global.azurebilling.image.repository` | csp adapter image repository | `neuvector/neuvector-csp-adapter` | Follow Azure subscription instruction
```

Comment:
If we do not want to maintain a seperate fork or branch for building the CNAB for Azure, we need to follow these guidelines.  https://learn.microsoft.com/en-us/partner-center/marketplace/azure-container-technical-assets-kubernetes?tabs=linux#update-the-helm-chart